README.md

```diff
@@ -27,13 +27,15 @@ kubectl create ns <namespace>
 cargo run --bin cluster --
     -n <namespace>
     --local-path <path-to-local-agave-monorepo>
+    --cluster-data-path <path-to-directory-to-store-cluster-accounts-genesis-etc>
```

Comment:
it can be an empty directory. it is just a directory to store client accounts, validator accounts, the ledger, and dockerfiles. I can update readme to clarify this.